controls/SV-257847.rb

```diff
@@ -15,15 +13,27 @@
   desc 'fix', 'Migrate the system audit data path onto a separate file system.'
   impact 0.3
   ref 'DPMS Target Red Hat Enterprise Linux 9'
-  tag check_id: 'C-61588r925526_chk'
   tag severity: 'low'
+  tag gtitle: 'SRG-OS-000341-GPOS-00132'
   tag gid: 'V-257847'
   tag rid: 'SV-257847r925528_rule'
   tag stig_id: 'RHEL-09-231030'
-  tag gtitle: 'SRG-OS-000341-GPOS-00132'
   tag fix_id: 'F-61512r925527_fix'
-  tag satisfies: ['SRG-OS-000341-GPOS-00132', 'SRG-OS-000480-GPOS-00227']
-  tag 'documentable'
   tag cci: ['CCI-000366', 'CCI-001849']
   tag nist: ['CM-6 b', 'AU-4']
+  tag 'host'
+
+  only_if('This control is Not Applicable to containers', impact: 0.0) {
+    !virtualization.system.eql?('docker')
+  }
+
+  audit_data_path = command("dirname #{auditd_conf.log_file}").stdout.strip
```

Comment:
i cannot find this auditd_conf input in the inspec.yml nor is it used in any of the sample/testing inputs.ymls.